src/main/java/io/connect/scylladb/ScyllaDbSessionFactory.java

```diff
@@ -101,4 +130,21 @@ public ScyllaDbSession newSession(ScyllaDbSinkConnectorConfig config) {
     final Session session = cluster.newSession();
     return new ScyllaDbSessionImpl(config, cluster, session);
   }
+
+  private KeyStore createKeyStore(File path, char[] password) {
+    KeyStore keyStore;
+    try {
+      keyStore = KeyStore.getInstance("JKS");
```

Comment:
I am not a fan of nested Try-Catch blocks especially without using finally, it just affects the readability of the code.